### PR TITLE
Go over operator characters...

### PR DESCRIPTION
**Pull request by rocky**
and add more information in operators YAML

### CODE REVIEW

File: mathics_scanner/data/named-characters.yml [hunk 1]
```diff
@@ -5003,13 +4992,15 @@ Information:
   is-letter-like: false
   operator-name: Information
 
+# While one might think that Integral is an WL operator, it is not.
```
Comment (mmatera) [inline mathics_scanner/data/named-characters.yml:4995]: > ### Precedence of Integration Operators
>
> Forms such as involve two operators: [\[Integral]](https://reference.wolfram.com/language/ref/ character/Integral.html) 
> and [\[DifferentialD]](https://reference.wolfram.com/language/ref/character/DifferentialD.html). 
> The former has a precedence just above , while the latter has one > just below [Power](https://reference.wolfram.com/language/ref/Power.html). 
> The precedence of determines when needs to be parenthesized, while the precedence of determines when needs to be parenthesized. Note that the form does not involve , and therefore behaves like a normal prefix operator in this case.
> [∮](https://reference.wolfram.com/language/ref/character/ContourIntegral.html), [∲](https://> reference.wolfram.com/language/ref/character/ClockwiseContourIntegral.html) and 
> [∯](https://reference.wolfram.com/language/ref/character/DoubleContourIntegral.html) work the same as [∫](https://reference.wolfram.com/language/ref/character/Integral.html).
> See ["Two-Dimensional Input Forms"](https://reference.wolfram.com/language/tutorial/InputSyntax.html#15273) for two‐dimensional input forms associated with integration operators.

(from https://reference.wolfram.com/language/tutorial/OperatorInputForms.html)

File: test/test_general_yaml_sanity.py [hunk 2]
```diff
@@ -180,7 +182,7 @@ def test_unicode_operators():
         operator_name = v["operator-name"]
         assert (
             k == operator_name
-        ), f"Section name {k} should match operator-name {operator_name} when a section has an operator"
+        ), f"Section name {k} should match operator-name {operator_name} or be exlicitly excluded when a section has an operator"
```
Comment (mmatera) [inline test/test_general_yaml_sanity.py:185]: explicitly?

File: mathics_scanner/data/operators.yml [hunk 3]
```diff
@@ -42,14 +46,34 @@
...
-
-
+#
+#
 #   actual-precedence: a precedence value that programs can use.
+#
 #   Precedence-corrected: the value used in Mathics3.
-#   WolframLangageData: a value returned using WolframLangageData[???]
+
+#   WolframLangageData: a "PrecedenceRanks" value returned using:
```
Comment (mmatera) [inline mathics_scanner/data/operators.yml:55]: WolframLanguageData

File: mathics_scanner/data/operators-intro.yml [hunk 4]
```diff
@@ -41,14 +45,34 @@
...
-
-
+#
+#
 #   actual-precedence: a precedence value that programs can use.
+#
 #   Precedence-corrected: the value used in Mathics3.
-#   WolframLangageData: a value returned using WolframLangageData[???]
+
+#   WolframLangageData: a "PrecedenceRanks" value returned using:
```
Comment (mmatera) [inline mathics_scanner/data/operators-intro.yml:54]: WolframLanguageData

File: mathics_scanner/data/named-characters.yml (hunk 1 above)
Comment (rocky) [reply mathics_scanner/data/named-characters.yml:4995]: Ok. Thanks for the information and corrections.

I will revise this and the spelling corrections below soon. The thing about Integral then is that its "arity" is special- neither Prefix, Postfix, Infix, or MatchFix. I'll create a new category "Integral" then.

File: test/test_general_yaml_sanity.py (hunk 2 above)
Comment (rocky) [reply test/test_general_yaml_sanity.py:185]: Yes  - thanks.

File: mathics_scanner/data/operators.yml (hunk 3 above)
Comment (rocky) [reply mathics_scanner/data/operators.yml]: Should be fixed now.

**Pull request merged**